README.md

```diff
@@ -1,6 +1,6 @@
 # No Code
 
-No code is the best way to write secure and reliable applications. Write nothing; deploy nowhere.
+No code is the best way to write secure and reliable applications. Write nothing; deploy nowhere. Now running on the JVM for added security, portability and reliability.
```

Comment:
This breaks in java 5. please fix, as pointed here: https://stackoverflow.com/questions/48565151/java-program-not-running-error-box-is-blank